src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts

```diff
@@ -0,0 +1,207 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  CSSResult,
+  css,
+} from "lit-element";
+import "@polymer/paper-tabs";
+
+import { struct } from "../../common/structs/struct";
+import { HomeAssistant } from "../../../../types";
+import { LovelaceCardEditor } from "../../types";
+import { StackCardConfig } from "../../cards/types";
+import { fireEvent } from "../../../../common/dom/fire_event";
+
+const cardConfigStruct = struct({
+  type: "string",
+  cards: ["any"],
+  title: "string?",
+});
+
+@customElement("hui-stack-card-editor")
+export class HuiAlarmPanelCardEditor extends LitElement
+  implements LovelaceCardEditor {
+  @property() public hass?: HomeAssistant;
+  @property() private _config?: StackCardConfig;
+  @property() private _selectedCard: number = 0;
+
+  public setConfig(config: StackCardConfig): void {
+    this._config = cardConfigStruct(config);
+  }
+
+  protected render(): TemplateResult {
+    if (!this.hass || !this._config) {
+      return html``;
+    }
+    const selected = this._selectedCard!;
+    const numcards = this._config.cards.length;
+
+    return html`
+      <div class="card-config">
+        <div class="toolbar">
+          <paper-tabs
+            selected="${selected}"
+            scrollable
+            @iron-select=${(ev) => (this._selectedCard = ev.target.selected)}
```

Comment:
`ev.target.selected` is a string, you have to convert it to a number. And please move this, no inline functions.